phalcon/config/adapter/ini.zep

```diff
@@ -58,11 +58,16 @@ class Ini extends Config
 	/**
 	 * Phalcon\Config\Adapter\Ini constructor
 	 */
-	public function __construct(string! filePath)
+	public function __construct(string! filePath, mode = null)
 	{
 		var iniConfig;
 
-		let iniConfig = parse_ini_file(filePath, true, INI_SCANNER_RAW);
+		// Default to INI_SCANNER_RAW if not specified
+		if null === mode {
```

Comment:
Could you please 
* update example in class header
* update CHANGELOG.md (with link to the issue)